nbviewer/handlers.py

```diff
@@ -45,12 +47,25 @@ class CreateHandler(BaseHandler):
 
     only redirects to the appropriate URL
     """
+    uri_rewrite_list = None
+
     def post(self):
         value = self.get_argument('gistnorurl', '')
-        redirect_url = transform_ipynb_uri(value, self.provider_rewrites)
+        redirect_url = transform_ipynb_uri(value, self.get_provider_rewrites())
         app_log.info("create %s => %s", value, redirect_url)
         self.redirect(redirect_url)
 
+    @classmethod
```

Comment:
good stuff. did something very similar over on ipymd, though I think it used a singleton instance.